web_widget_html_markdown/README.rst

```diff
@@ -0,0 +1,131 @@
+========================================
+Web Widget Html Markdown Unidirectional 
+========================================
```

Comment:
This title is fetched from manifest. I gave it this name , in the hopes of a 2-way version. But i will remove. 
If we make a 2-way version it will be a replacement. cleaning up.

Reply:
Done